Mathlib/SetTheory/Ordinal/Basic.lean

```diff
@@ -1145,7 +1145,8 @@ theorem gc_ord_card : GaloisConnection ord card := fun _ _ => ord_le
 theorem lt_ord {c o} : o < ord c ↔ o.card < c :=
   gc_ord_card.lt_iff_lt
 
-theorem pos_iff_ord_pos {c : Cardinal} : 0 < c ↔ 0 < c.ord := by
```

Comment:
Can you add a deprecation here?

Reply:
But this is a theorem I added in this PR

Reply:
Oops, my bad.

Reply:
You just can't stop deprecating things, can you? 😔